src/Catalog/Monitoring/Validation/Test/ValidationTestRunData.cs

```diff
@@ -0,0 +1,49 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System.Collections.Generic;
+using System.Threading;
+using NuGet.Packaging.Core;
+using NuGet.Services.Metadata.Catalog.Helpers;
+using Microsoft.Extensions.Logging;
+
+namespace NuGet.Services.Metadata.Catalog.Monitoring.Validation.Test
+{
+    public class ValidationTestRunData
```

Comment:
This is a common pattern for methods with many parameters. It could be called "Request" or "Context" instead of "Data" to allude to this. Client code has many *Context types.